src/features/collection/state/collectionActions.ts

```diff
@@ -15,6 +16,18 @@ export function loadCollection() {
   }
 }
 
+export function loadVersionInfo(initID: string) {
+  return async (dispatch, getState: () => RootState) => {
+    const state = getState();
+    if(state.collection.collection[initID]){
+      return state.collection.collection[initID]
+    }else if(!state.collection.pendingIDs.includes(initID)){
+      return dispatch(fetchVersionInfo(initID))
+    }else return
```

Comment:
some linting errors here:

```
    if (state.collection.collection[initID]) {
      return state.collection.collection[initID]
    } else if (!state.collection.pendingIDs.includes(initID)) {
      return dispatch(fetchVersionInfo(initID))
    } else return
```

Reply:
_still_ missing. Maybe the wrong rule is writing on save in your IDE?

Reply:
@b5 ,  sorry.. fixed ^